okapi-core/src/main/raml/okapi.raml

```diff
@@ -790,10 +815,18 @@ types:
           must be deployed (optionally) and enabled/disabled by the existing
           tenants-modules CRUD service.
         queryParameters:
+          async:
+            description: Whether to install in the background
+            type: boolean
+            required: false
           deploy:
             description: Whether to deploy (or undeploy if disabling)
             type: boolean
             required: false
+          ignoreErrors:
+            description: Whether to ignore module init errors
+            type: boolean
+            required: false
```

Comment:
The default value is missing. We need it to display on https://dev.folio.org/reference/api/#okapi

Reply:
OK